### PR TITLE
Replicate the same structure as on status.im

### PR DESCRIPTION
**Pull request by santteegt**
## Scenario

Nimbus website needs to replicate the same structure as on status.im, so it is needed to change all `.styl` files to `.scss`, and update `.swig` layouts and JS scripts accordingly.

This PR aims to solve bounty proposed in #20 

## Changes proposed in this pull request:

1. Remove all unnecessary styles and javascript files.
1. Change all `.styl` files to `.scss` files.
1. Use the same gulpfile as in that status.im to make sure all scss, js, and images are minified.
1. Replicate the same main menu as on the main site.
1. Update the README to reflect the gulp changes.

### CODE REVIEW

**Comment by StatusSceptre** [comment]
Hey @santteegt - thanks ❤️ First question after cloning your branch: how could you have tested this PR locally if you haven't edited the `package.json` to use gulp?

File: themes/navy/source/scss/projects.scss [hunk 1]
```diff
@@ -0,0 +1,222 @@
+.cards--projects a.card-inner {
```
[the remark this answers is not in the document]
Comment (StatusSceptre) [reply themes/navy/source/scss/projects.scss]: nm, I found the gulpfile, git was just being weird in Africa for a moment

File: themes/navy/layout/partial/header.swig [hunk 2]
```diff
@@ -1,78 +1,169 @@
...
+            <li class="item--to-show"> <a href={{url_for_lang('docs')}}>Docs</a></li>
+            <li class="item--dropdown item--dropdown-projects"> <a href="#" class="">Projects</a></li>
+            <li><a href="https://status.im/contribute/">Contribute</a></li>
+            <li class="item--dropdown"><a href="#" class="item--dropdown-community">Community</a></li>
+            <li class="item--more"><a href="#" class="item--more">More</a></li>
+        </ul>
+    </div>
+    <div class="header-right">
+        <a href="http://get.status.im/chat/public/status-nimbus" class="button button--secondary" style="margin-right: 8px">Nimbus Status Chat</a>
+        <span class="social-link social-link--twitter"><a href="https://twitter.com/ethstatus" target="_blank"></a></span>
```
Comment (StatusSceptre) [inline themes/navy/layout/partial/header.swig]:
```suggestion
```

File: themes/navy/layout/partial/header.swig [hunk 3]
```diff
@@ -1,78 +1,169 @@
...
+        <span class="social-link social-link--github"><a href="https://github.com/status-im/nimbus" target="_blank"></a></span>
```
Comment (StatusSceptre) [inline themes/navy/layout/partial/header.swig]:
```suggestion
```

**Comment by StatusSceptre** [comment]
Otherwise, this is really awesome work @santteegt thank you ❤️ Make these minor fixes and we can merge and payout here.

File: themes/navy/layout/partial/header.swig (hunk 3 above)
Comment (StatusSceptre) [reply themes/navy/layout/partial/header.swig]: Also remove the button below that, so it's just the Nimbus chat in Status button in the top right.

File: themes/navy/source/scss/projects.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (santteegt) [reply themes/navy/source/scss/projects.scss]: You're right. Removing this SCSS file

**Pull request merged**